api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java

```diff
@@ -22,6 +25,107 @@
  */
 public interface CBCriteriaDao extends CrudRepository<DbCriteria, Long>, CustomCBCriteriaDao {
 
+  /** Consolidate all Survey Question Dao calls into one default method. */
+  default Page<DbCriteria> findSurveyQuestions(
```

Comment:
added 5 default methods in CBCriteriaDao to allow us to push which dao call happens into this layer and cleanup the service layer.